en/app.ftl

```diff
@@ -24,6 +24,8 @@
        *[lowercase] Firefox account
         [uppercase] Firefox Account
     }
+-brand-name-chrome = Chrome
+-brand-name-google-chrome = Google Chrome
```

Comment:
@peiying2 I'm not sure if we're supposed to create variables for all brand names, or just the ones that are our own brands?

Reply:
Yes, we do. mozilla.org site/project contains tons of 3rd party brand/product names. Sticking to the same practice will prevent them from being translated or if is, the term stays consistent.